e2e-samples/GitIntegration-ConnectAndUpdateFromGit.ps1

```diff
@@ -0,0 +1,156 @@
+# This sample script calls the Fabric API to programmatically connect to and update the workspace with content in Git.
+
+# For documentation, please see:
+# https://learn.microsoft.com/en-us/rest/api/fabric/core/git/connect
+# https://learn.microsoft.com/en-us/rest/api/fabric/core/git/initialize-connection
+# https://learn.microsoft.com/en-us/rest/api/fabric/core/git/update-from-git
+
+# Instructions:
+# 1. Install PowerShell (https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell)
+# 2. Install Azure PowerShell Az module (https://learn.microsoft.com/en-us/powershell/azure/install-azure-powershell)
+# 3. Run PowerShell as an administrator
+# 4. Fill in the parameters below
+# 5. Change PowerShell directory to where this script is saved
+# 5. > ./GitIntegration-ConnectAndUpdateFromGit.ps1
+
+# Parameters - fill these in before running the script!
+# =====================================================
+
+$workspaceName = "FILL ME"      # The name of the workspace
+
+# The Git provider details
```

Comment:
This is an ADO specific sample.
For other Git providers (like GitHub) we'll have different params